tests/PHPStan/Type/ObjectTypeTest.php

```diff
@@ -135,7 +135,7 @@ public function dataIsSuperTypeOf(): array
 			12 => [
 				new ObjectType(\InvalidArgumentException::class),
 				new ObjectType(\LogicException::class),
-				TrinaryLogic::createMaybe(),
+				TrinaryLogic::createNo(),
```

Comment:
This is also wrong.  InvalidArgumentException extends LogicException so: objects that extend LogicException might also extend InvalidArgumentException is "maybe" is the correct answer.

If those two classes were unrelated to each other (or only had a common parent like `Exception`), then "no" would be correct.

If you're still unsure why is that, I can explain this to you over DM if you want.